domain/model/student.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+type Student struct {
```

Comment:
By convention, all the descriptions should start with the name of the struct, interface, function, etc name
Example:
`// NameOfTheStruct - Description`

Reply:
Done